code/modules/client/preference_setup/loadout/lists/misc.dm

```diff
@@ -18,7 +18,6 @@
 /datum/gear/clown
 	display_name = "clown pack"
 	path = /obj/item/storage/box/clown
-	cost = 3
```

Comment:
Doesn't this pack give multiple powerful items such as a banana peel?

Reply:
oh. my. god. clown meta incoming??? literally unplayable

Reply:
Iirc it does but I dont think that's super strong? 
I don't play clown but I've never had an issue finding a slip item, and 3 points for a funny gimmick seems too much.